src/throughputmanagement.c

```diff
@@ -179,6 +179,21 @@ void run_ntttcp_throughput_management(struct ntttcp_test_endpoint *tep)
 			break;
 	}
 
+	/* Wait for stream processing completion before capturing statistics */
```

Comment:
Thanks for the code.

This code will break the scenario of testing with cooldown ('-C'). If -C specified in sender, the bytes collected from sender side include bytes from cooldown stage, which is incorrect.